### ISSUE TITLE
README update - os-proxy only supports Mac

### ISSUE THREAD

**Issue by gannons** [sholladay/os-proxy#6]
My apologies in advance for critiquing someone offering software free of charge online. But the statement

> Cross-platform behavior.

is misleading given os-proxy only supports Mac. i.e.

https://github.com/sholladay/os-proxy/blob/2bfbb9ed92b7b6b2be9661fce3b03a2cc2fb8c85/index.js#L17

**Comment by sholladay** [comment]
You're right. Being cross-platform is a goal that I thought I could get around to very quickly, but it never happened.

A PR is welcome. Otherwise, I'll eventually remove that or work on cross-platform support.
